akka-actor/src/main/scala/akka/io/dns/internal/DnsClient.scala

```diff
@@ -82,21 +82,21 @@ import scala.concurrent.duration._
     case Question4(id, name) =>
       log.debug("Resolving [{}] (A)", name)
       val msg = message(name, id, RecordType.A)
-      inflightRequests += (id -> (sender(), msg))
+      inflightRequests += (id -> ((sender(), msg)))
```

Comment:
is this needed if written as `(sender() -> msg)`?

Reply:
better

Reply:
That's indeed nicer, thx!